autotx/utils/coingecko/api.py

```diff
@@ -0,0 +1,17 @@
+import json
+import os
+
+import requests
+
+
+class CoingeckoApi:
```

Comment:
Instead of implementing this ourselves, we could use a library like this: https://github.com/khooihzhz/coingecko-python

It's decently typed, and has a good separation between basic & pro features.

Reply:
sure - i just wanted to avoid adding more 3rd party dependencies until we add the `mypy` script because it's already troubling :yum: